src/resize-sensor/ResizeSensor.d.ts

```diff
@@ -0,0 +1,14 @@
+import { PureComponent } from 'react';
+import { Type } from 'tcomb';
+
+export type ResizeSensorProps = {
+  children: any,
+  onResize: (elements: HTMLElement) => void,
```

Comment:
I digged a little into `css-element-queries` and found https://github.com/marcj/css-element-queries/blob/master/src/ResizeSensor.js#L52